website/content/commands/peering/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+layout: commands
+page_title: 'Commands: Peering'
+---
+
+# Consul Peering
+
+Command: `consul peering`
+
+The `peering` command provides management of cluster peering connections, including generating and
+consuming peering tokens.
+[Create and Manage Peerings Connections](/docs/connect/cluster-peering/create-manage-peering) provides an
+overview of the CLI workflow for cluster peering.
+This was added in Consul 1.14.
```

Comment:
```suggestion
Use the `peering` command to create and manage peering connections between Consul clusters, including token generation and consumption. Refer tp 
[Create and Manage Peerings Connections](/docs/connect/cluster-peering/create-manage-peering) for an
overview of the CLI workflow for cluster peering.
```
I don't think we need to include the version since we can control which version of the docs the information appears in.

Reply:
Cool! Using this suggestion to fix the "to" typo
```suggestion
Use the `peering` command to create and manage peering connections between Consul clusters, including token generation and consumption. Refer to 
[Create and Manage Peerings Connections](/docs/connect/cluster-peering/create-manage-peering) for an
overview of the CLI workflow for cluster peering.
```